zmq_addon.hpp

```diff
@@ -37,6 +37,78 @@
 
 namespace zmq
 {
+
+#ifdef ZMQ_CPP11
+
+/*  Receive a multipart message.
+    
+    Writes the zmq::message_t objects to OutputIterator out.
+    The out iterator must handle an unspecified amount of write,
```

Comment:
```suggestion
    The out iterator must handle an unspecified number of writes,
```
See also comment below on throwing.